### ISSUE TITLE
Improve preset categories colors

### ISSUE THREAD

**Issue by jjiglesiasg** [openstreetmap/iD#8799]
### Description

It seems logic to me that the menu colours should be in accordance with what elements are depicted. As such the Menu folder for Waterways should be Blue, and Natural features should be green or brown, but not Blue wich is traditionally associated with Water Features that currently has No colours.

### Screenshots

![Screenshot 2021-11-09 18 58 17](https://user-images.githubusercontent.com/22849578/141019344-141cfed9-c153-49e8-be45-577d3721cde3.png)
